arch/arm64/boot/dts/rockchip/overlays/radxa-zero3-waveshare2-lcd.dts

```diff
@@ -0,0 +1,39 @@
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/rockchip.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+
+/ {
+	metadata {
+		title = "Enable Waveshare 2inch LCD Display Module";
+		compatible = "radxa,zero3";
+		category = "display";
+		exclusive = "GPIO4_C3", "GPIO4_C2", "GPIO4_C6", "GPIO3_C1", "GPIO3_B2", "GPIO3_A2", "spi3";
+		description = "Enable Waveshare 2inch LCD Display Module.";
+	};
+};
+
+&spi3{
+	status = "okay";
+	pinctrl-names = "default", "high_speed";
+	pinctrl-0 = <&spi3m1_cs0 &spi3m1_pins>;
+	pinctrl-1 = <&spi3m1_cs0 &spi3m1_pins_hs>;
+
+	st7789v@0 {
+		compatible = "sitronix,st7789v";
+		reg = <0>;
+		spi-max-frequency = <50000000>;
+		rotate = <90>;
+		fps = <60>;
+		buswidth = <8>;
+		regwidth = <8>;
+		width = <240>;
+		height = <320>;
+		cs-gpio = <&gpio4 RK_PC6 GPIO_ACTIVE_HIGH>;
```

Comment:
```suggestion
```
This is not needed, when the same pin also supports hardware SPI CS function. This is only for other non-SPI GPIO pin to emulate the CS function.